### DISCUSSION TITLE
Wikipedia needs a page for this project

### DISCUSSION THREAD

**Discussion by ankostis** [nushell/nushell#13834] [General]
I didn't manage to find a dedicated Wikipedia page for this beautiful project.
Apart from this, the following pages are relevant:

- https://en.wikipedia.org/wiki/Comparison_of_command_shells
- https://en.wikipedia.org/wiki/Template:Unix_shells
- https://en.wikipedia.org/wiki/Category:Unix_shells
- https://en.wikipedia.org/wiki/Unix_shell
- https://en.wikipedia.org/wiki/Template:Infobox_software

**Comment by NotTheDr01ds** [comment]
This has come up before, in https://github.com/nushell/nushell.github.io/issues/321, but I haven't commented on it there yet.  I've noodled on it on and off, but since you ask, I'll go ahead and throw out my thoughts:

* Keep in mind that [Wikipedia requires that topics have attained a certain level of "Notability"](https://en.wikipedia.org/wiki/Wikipedia:Notability) before articles can be created about them.  I'm not sure that we are quite to the point of having the "significant coverage", over "a significantly significant period of time", required yet.  Maybe we are, but at this stage there are only two Nushell-specific YouTube videos with more than 10k views.  Random Hacker News blog submissions probably can't be considered "significant".

* Notability guidelines don't apply to items in lists, so we can certainly add it to those existing lists, such as the Comparisons page.  Perhaps I'll take a stab at that at some point.  It's a pretty big comparison list ;-), but I think we compare fairly favorably there.

**Comment by NotTheDr01ds** [comment]
Although taking a second look at the Comparisons page, all of those have a top-level link, even though some should probably be removed under the Notability guidelines.

That's honestly one convoluted comparison, and I'm not sure how useful it is to have so many defunct (or nearly defunct) shells in there from 20+ years ago.

**Comment by ankostis** [comment]
I'd say the notability is guaranteed.  Crafting a main page with [a Infobox software template](https://en.wikipedia.org/wiki/Template:Infobox_software) should be enough to start this activity.
